src/backend/base/langflow/components/data/api_request.py

```diff
@@ -118,14 +124,15 @@ class APIRequestComponent(Component):
                     "description": "Header value",
                 },
             ],
-            value=[],
+            value=[{"key": "User-Agent", "value": user_agent}],
```

Comment:
Could you do something like this so just so we don't keep the variable in memory?

```suggestion
            value=[{"key": "User-Agent", "value": get_settings_service().settings.user_agent}],
```